libs/facilities/frontend/view/src/lib/components/browse/browse.page.ts

```diff
@@ -1,13 +1,36 @@
 import { CommonModule } from '@angular/common';
-import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/core';
+import {
+	ChangeDetectionStrategy,
+	Component,
+	Input,
+	OnInit,
+	ViewEncapsulation,
+} from '@angular/core';
+import { RouterLink } from '@angular/router';
+import { IxModule } from '@siemens/ix-angular';
+
+import { facilities, Facility } from '../facility';
 
 @Component({
 	selector: 'lib-browse',
 	standalone: true,
-	imports: [CommonModule],
+	imports: [CommonModule, IxModule, RouterLink],
+
 	templateUrl: './browse.page.html',
 	styleUrl: './browse.page.scss',
 	encapsulation: ViewEncapsulation.None,
 	changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class XdBrowsePage {}
+export class XdBrowsePage implements OnInit {
+	@Input() facilities: Facility[] = facilities;
+	@Input() subtitle = 'List of all facilities';
+
+	ngOnInit() {
+		if (this.facilities === undefined) {
+			this.facilities = facilities;
+		}
+		if (this.subtitle === undefined) {
+			this.subtitle = 'List of all facilities';
+		}
```

Comment:
Not neccessary no fallback values. If u want to do sth like that in the future use https://stackoverflow.com/questions/36653678/angular2-input-to-a-property-with-get-set

Reply:
Before I added ngOnInit the values were undefined. Might do it better in the future